pyseir/inference/model_fitter.py

```diff
@@ -631,6 +631,8 @@ def fit(self):
                 f"Epsilon == 0 which implies lack of convergence."
             )
 
+        # For now we are not applying posterior updating after the fit, but trust the fit to find the best R values
+        """
```

Comment:
Comment out each line with #. That's for docstrings and makes all these compare tools hard to use.